web_src/js/index.js

```diff
@@ -1279,6 +1279,22 @@ async function initRepository() {
       $('.language-stats-details, .repository-menu').slideToggle();
     });
   }
+
+  // custom repo buttons
+  (function() {
+    if ($('.repo-buttons').length === 0) {
+      return;
+    }
+
+    const $detailModal = $('#detail-modal');
+
+    $('.show-repo-button-content').on('click', function () {
```

Comment:
I think this would be a good case for @silverwind's jsx PR, as I am concerned with the below line of injecting HTML and it creating an XSS issue.

Reply:
@silverwind How about current way? Thanks

Reply:
Better, thanks.